tests/integration_python/build/test_build.py

```diff
@@ -44,3 +45,92 @@ def test_build_conda_package(pixi: Path, tmp_path: Path) -> None:
     package_to_be_built = next(manifest_path.parent.glob("*.conda"))
 
     assert package_to_be_built.exists()
+
+
+def test_build_using_rattler_build_backend(pixi: Path, tmp_path: Path) -> None:
+    manifest_path = tmp_path / "pixi.toml"
+
+    # Create a new project
+    verify_cli_command([pixi, "init", tmp_path])
+
+    parsed_manifest = tomllib.loads(manifest_path.read_text())
+    parsed_manifest["host-dependencies"] = {"hatchling": "*"}
+    parsed_manifest["build"] = {
+        "build-backend": "pixi-build-rattler-build",
+        "channels": [
+            # until rattler-build backend is merged in pixi build backend,
```

Comment:
I think this should be merged actually :)

https://prefix.dev/channels/pixi-build-backends/packages/pixi-build-rattler-build

Also graf cannot ever run on any other platform then arm-osx, so cant really merge then :)

Reply:
changed!